docs: :memo: moved content from README to landing page (#116)
seedcase-project/template-data-package: index.qmd

following the Seedcase structure. Use this template to quickly get
started with a new Data Package with all the necessary files and
configurations in place, including for developing the Data Package.

## Features

::: content-hidden
<!-- TODO: Move this down once a license has been decided on. -->

-   Licensed under the open data license TODO. This is added by default,
    but you can delete or modify it if it isn't what you want or can
    use.
:::

-   Use [uv](https://decisions.seedcase-project.org/why-uv/) to manage
    the Data Package.
-   Use [Polars](https://www.pola.rs/) for data manipulation.
-   Use [Seedcase Sprout](https://sprout.seedcase-project.org/) to
    create and modify the Data Package.
-   Build management with [justfile](https://just.systems/man/en/).
-   Check typos with [typos](https://github.com/crate-ci/typos).
-   Code formatting and linting with
    [ruff](https://decisions.seedcase-project.org/why-ruff/).
-   Pre-commit hooks with [pre-commit](https://pre-commit.com/).
-   GitHub workflows for automatically adding Pull Requests and Issues
    to a project board.
-   Pull Request template for easy creation of new Pull Requests.

> -   [VS Code](https://code.visualstudio.com/) settings for common
>     settings for contributors using VS Code.

Reviewer: ```suggestion
-   [VS Code](https://code.visualstudio.com/) settings commonly
    used by contributors working in VS Code.
```
